imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -3879,3 +3879,50 @@ func requireOpenFile(t *testing.T, tmpDir string, pathName string, data []byte,
 
 	return mod, fd, log, r
 }
+
+// Test_fdReaddir_opened_file_written ensures that writing files to the already-opened directory
+// is visible. This is significant on Windows.
+// https://github.com/ziglang/zig/blob/2ccff5115454bab4898bae3de88f5619310bc5c1/lib/std/fs/test.zig#L156-L184
+func Test_fdReaddir_opened_file_written(t *testing.T) {
```

Comment:
can you add this test to dirfs_test or move it? usually I just convert it. that way we get it covered when implementing the next wasi. I don't think this is wasi-specific, even if it was noticed in wasi.